Apps/W1/EDocumentsTietoevryConnector/app/src/Tietoevry/Cod96360.TietoevryConnectionEvents.al

```diff
@@ -0,0 +1,113 @@
+namespace TietoevryConnector.TietoevryConnector;
+
+using Microsoft.Sales.Peppol;
+using Microsoft.Sales.Document;
+using Microsoft.Foundation.Company;
+using Microsoft.Sales.Customer;
+using Microsoft.EServices.EDocumentConnector;
+using Microsoft.eServices.EDocument;
+using System.Utilities;
+
+codeunit 96360 "Tietoevry Connection Events"
+{
+    SingleInstance = true;
+    EventSubscriberInstance = StaticAutomatic;
+
+    [EventSubscriber(ObjectType::Codeunit, Codeunit::"PEPPOL Validation", OnCheckSalesDocumentOnBeforeCheckCompanyVATRegNo, '', false, false)]
+    local procedure "PEPPOL Validation_OnCheckSalesDocumentOnBeforeCheckCompanyVATRegNo"(SalesHeader: Record "Sales Header"; CompanyInformation: Record "Company Information"; var IsHandled: Boolean)
+    var
+        ExternalConnectionSetup: Record "Tietoevry Connection Setup";
+    begin
+        if not ExternalConnectionSetup.Get() then
+            Error(MissingSetupErr);
+        IsHandled := true;
+        if CompanyInformation."VAT Registration No." = '' then
+            Error(MissingCompInfVATRegNoErr, CompanyInformation.TableCaption());
+    end;
+
+    [EventSubscriber(ObjectType::Codeunit, Codeunit::"PEPPOL Validation", OnCheckSalesDocumentOnBeforeCheckCustomerVATRegNo, '', false, false)]
+    local procedure "PEPPOL Validation_OnCheckSalesDocumentOnBeforeCheckCustomerVATRegNo"(SalesHeader: Record "Sales Header"; Customer: Record Customer; var IsHandled: Boolean)
+    begin
+        IsHandled := true;
+        if (SalesHeader."Document Type" in [SalesHeader."Document Type"::Invoice, SalesHeader."Document Type"::Order, SalesHeader."Document Type"::"Credit Memo"]) and
+               Customer.Get(SalesHeader."Bill-to Customer No.")
+            then
```

Comment:
The check on the customer and the VAT registration no. doesn't look correct.
At least this doesn't look right together with the new Multiple VAT Registration No. feature.

Reply:
Pattern (use of code) was forked from your Pagero implementation of E-Document Connector.

Reply:
bump?

Reply:
I'm no native english person and I don't know what you mean by bump.

My origional question was about the check of the VAT Registration No. instead of the VAT Registration No. of the Sales Document.
I asked the same now in yammer.
https://www.yammer.com/dynamicsnavdev/threads/2969533675659264